spec/fixtures/app/components/f/package.json

```diff
@@ -3,5 +3,8 @@
   "version": "0.0.1",
   "description": "Some description",
   "author": "Garett Arrowood",
-  "license": "UNLICENSED"
+  "license": "UNLICENSED",
```

Comment:
This really *Unlicensed*? Not relevant to this PR but something we should address at some point.